From dcaa09ae25677bc0f49604cc04f5a81e7d800765 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Tue, 30 Jul 2013 13:05:00 +0800
Subject: [PATCH] ENGR00256988 Wifi:change compile rules

This is qcom's newest private codes for wpa_supplicant.
Change rule to compile it for UNITE device.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 qcwcn/Android.mk                      | 2 +-
 qcwcn/wpa_supplicant_8_lib/Android.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/qcwcn/Android.mk b/qcwcn/Android.mk
index a8fc4b3..b83e42b 100644
--- a/qcwcn/Android.mk
+++ b/qcwcn/Android.mk
@@ -1,3 +1,3 @@
-ifeq ($(BOARD_WLAN_DEVICE),qcwcn)
+ifeq ($(BOARD_WLAN_DEVICE),$(filter $(BOARD_WLAN_DEVICE), qcwcn UNITE))
     include $(call all-subdir-makefiles)
 endif
diff --git a/qcwcn/wpa_supplicant_8_lib/Android.mk b/qcwcn/wpa_supplicant_8_lib/Android.mk
index df715ae..cb7d174 100644
--- a/qcwcn/wpa_supplicant_8_lib/Android.mk
+++ b/qcwcn/wpa_supplicant_8_lib/Android.mk
@@ -15,7 +15,7 @@
 #
 LOCAL_PATH := $(call my-dir)
 
-ifeq ($(WPA_SUPPLICANT_VERSION),VER_0_8_X)
+ifeq ($(WPA_SUPPLICANT_VERSION),$(filter $(WPA_SUPPLICANT_VERSION),VER_0_8_X VER_0_8_UNITE))
 
 ifneq ($(BOARD_WPA_SUPPLICANT_DRIVER),)
   CONFIG_DRIVER_$(BOARD_WPA_SUPPLICANT_DRIVER) := y
-- 
1.8.0

